frontend/ui/bidi.lua

```diff
@@ -283,4 +284,18 @@ function Bidi._filepath_rtl(path)
     return Bidi.ltr(Bidi._path(dirpath) .. Bidi._filename_auto_ext_right(filename))
 end
 
+function Bidi.invert()
```

Comment:
These deserve to be in the top half of bidi.lua with the related _mirrored_ui_layout stuff (the bottom half is about wrapping RTL text).

Reply:
Done